modules/c++/sys/wscript

```diff
@@ -74,16 +67,7 @@ def configure(conf):
                 elif v == 'False':
                     v = False
             conf.define(k.upper(), v)
-	output11 = conf.check_cxx(fragment = cpp11_str, execute=1, msg='Checking for C++11 support', define_ret=True, mandatory=False)
-	t2 = Utils.str_to_dict(output11 or '')
-	for k, v in t2.iteritems():
-		v = v.strip()
-		if v == 'True':
-			v=True
-		elif v == 'False': 
-			v=False;
-		conf.define(k.upper(), v)
-        
+        conf.check_cxx(fragment = cpp11_str, execute=1, msg='Checking for C++11 support', define_name='__CPP11', mandatory=False)
```

Comment:
I must have missed this part the first time... let's make the define name something like CODA_CPP11.  We want to make sure we don't conflict with some reserved preprocessor thing.